docs/api/rest_api/rest_api_reference/input/examples/ai/actions/GET/ActionConfiguration.xml.example

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ActionConfiguration media-type="application/vnd.ibexa.api.ai.ActionConfiguration+xml">
```

Comment:
I would regenerate this response as there were some changes to names, specifically lowercase attribute names